File: v2/snippets/testResources/net-dll/TestClass.cs
namespace NetcoreTestClass
{
    public class NetcoreTestClass
    {
		public int PublicValue { get; set; }
		private int PrivateValue { get; set; }
		public static int StaticValue = 3;

		public NetcoreTestClass()
        {
            PublicValue = 1;
            PrivateValue = 2;
        }

        public NetcoreTestClass(int publicValue, int privateValue)
        {
            PublicValue = publicValue;
            PrivateValue = privateValue;
        }

        public static int MultiplyByTwo(int a)
        {
            return 2 * a;
        }

		public int MultiplyTwoNumbers(int a, int b)
		{
			return a*b;
		}
	}
}

File: v2/snippets/testResources/nodejs_package/TestClass.js
class V8TestClass {

    publicValue;
    #privateValue;
    
    constructor(first, second) {
        this.publicValue = first;
        this.#privateValue = second;
    }

    static multiplyByTwo(value) {
        return value * 2;
    }

    static staticValue = 3;

    getPublicValue() {
        return this.publicValue;
    }

    setPublicValue(publicValue) {
        this.publicValue = publicValue;
    }

    getPrivateValue() {
        return this.#privateValue;
    }

    setPrivateValue(privateValue) {
        this.#privateValue = privateValue;
    }

    multiplyTwoNumbers(first, second) {
        return first*second
    }
}


module.exports = V8TestClass